.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules
 lerna-debug.log
 .eslintcache
 user-build-config.yml
+**/yarn-error.log
```

Comment:
Assuming we never want to push logs, we can replace `lerna-debug.log` and `yarn-error.log` with `*.log`.